public/partials/buyer_profile_large.html

```diff
@@ -6,17 +6,17 @@
                     <div layout="row">
                         <div flex="10" class="profilePic"></div>
                         <div flex="70" layout="column" class="farmerNames farmerPretty2">
+                            <label>Buyer</label>
 
                             <h2 class="farmerSummary">{{buyer.bu_buyer_name}}</h2>
-
                         </div>
                         <div layout="column" flex="20">
                             <div layout="row" >
                                 <md-button ng-click="editBuyer()" class="md-icon-button" aria-label="Edit">
                                     <md-icon md-svg-icon="/images/ic_mode_edit_24px.svg"></md-icon>
                                 </md-button>
-                                <md-button ng-click="createBuyerCall();"class="md-icon-button" aria-label="New Call">
```

Comment:
@nick22891 and @tremainebuchanan - This doesn't seem right... why are you removing the call button and it's function and replacing it with a favourites button?